### PR TITLE
Pip locally installable pygor module.

### PR DESCRIPTION
**Pull request by penuts7644**
Three changes were made:
- Added setup.py for local pip installation of pygor module. Changes were made to the init files within the pygor directories and import changes to all the python files.
- Updated the documentation to include this change (IGoR/docs/asciidoc/general/python_interface.adoc). Pygor can be installed using `pip install ./pygor` in the IGoR root directory.
- Syntax related changes applied to various parts withing the code to make it more PEP8 compliant.

I haven't made changes to the code itself. Also good to know that I used your (@qmarcou) contact information in the `setup.py` file.

### CODE REVIEW

**Comment by qmarcou** [comment]
Thank you so much for your help @penuts7644 ! 
I've been through your changes everything seems to work like a charm except for one change in your last commit about the 'Mismatch' vs 'Error' column in the bestscenarios module. See my comment inplace.
If you don't mind, I think it would be simpler and more transparent that you edit your PR rather than me accepting it and making the change again on top of it.
My apologies for the delay in my response and proper check, again many thanks!

File: pygor/pygor/counters/bestscenarios/bestscenarios.py [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+                                                    sep=","))
+
+            else:
+                tmp_real_indices = best_scenarios[event.name].apply(
+                    get_real_str_as_int)
+            best_scenarios_real[event.name] = tmp_real_indices.apply(
+                lambda x: real_vect[x])
+
+            # Return mismatches as a list of positions
+            best_scenarios_real["Errors"] = best_scenarios[
```
Comment (qmarcou) [inline pygor/pygor/counters/bestscenarios/bestscenarios.py]: Although you are right that using the `Mismatch` column was bogus to read the generated sequences realizations this does not completely solve the problem:
- the generated sequences realizations file contains an 'Error' field as by generating them one knows that there was an actual error or mutation at that place
- the assigned best scenarios contains a 'Mismatch' field instead and this column designates deviation between the scenario and the sequence.
The difference is subtle and probably not worth the distinction, however for a matter of retro compatibility I would rather edit this line as follows:
```python
if best_scenarios_real.columns.contains("Errors"): 
    best_scenarios_real["Errors"] = best_scenarios[
    "Errors"].apply(lambda x: utils.get_str_asarray(
                        x, dtype=int, boundaries_char=["(", ")"], sep=","))

if best_scenarios_real.columns.contains("Mismatches"): 
    best_scenarios_real["Mismatches"] = best_scenarios[
    "Mismatches"].apply(lambda x: utils.get_str_asarray(
                        x, dtype=int, boundaries_char=["(", ")"], sep=","))
```

Comment (penuts7644) [reply pygor/pygor/counters/bestscenarios/bestscenarios.py]: Implemented your fix! Thanks for the explanation @qmarcou

**Comment by qmarcou** [comment]
Thanks for the swift change! I'll close and merge this PR now, thanks a lot for your help!

**Pull request merged**